lua/nvim-tree/actions/fs/remove-file.lua

```diff
@@ -57,13 +57,18 @@ local function remove_dir(cwd)
   end
 
   while true do
-    local name, t = vim.loop.fs_scandir_next(handle)
+    local name, _ = vim.loop.fs_scandir_next(handle)
     if not name then
       break
     end
 
     local new_cwd = utils.path_join { cwd, name }
-    if t == "directory" then
+
+    -- Type must come from fs_stat and not fs_scandir_next to maintain sshfs compatibility
+    local stat = vim.loop.fs_stat(new_cwd)
```

Comment:
Should this be fs_lstat?

Reply:
I've found that making it `fs_lstat` made remove not update the tree view. Maybe a bug elsewhere?